exercises/strings/longestWord/longestWord.js

```diff
@@ -7,15 +7,50 @@
  * @returns {string} The longest word in the input string
  */
 
+/** arian's plan
+ * i'm not sure if i need to turn a string into an array in order to fit with the case jesse put
+ * go through the string and essentially every word
+ * determine the length of each word
+ * select the word with the highest number of characters
+ * return such word 
+ */
+
+
 function longestWord(string) {
-  // This is your job. :)
+  let stringToArray = string.split(' '); // doing this based on google searching... split string on space space
+  let longestSoFar = stringToArray; 
```

Comment:
You want to keep track of the longest string you've seen so far. Looking at each word, you compare the current word you're looking at to the longest you've seen so far.

If the current word is longer than the longest so far, you make the current word the longest so far. If the current word is not longer than the longest so far, you do nothing.

You continue this, one word at a time, until you've looked at every word. At that point, `longestSoFar` will contain the longest word in the entire list of words.

You do not want to initialize `longestSoFar` to the entire array of words. In your mind, it should only ever contain a string.

Since the empty string `''` is shorter than any other possible word, it makes sense to initialize it with the empty string. That is, *any other word* you see will be longer than `''`.